### PR TITLE
CON-428 INF try to workaround disk contention for content node

### PR DESCRIPTION
**Pull request by joaquincasares**
The idea for this PR came when reading these sections:

* https://docs.docker.com/config/containers/logging/configure/#configure-the-delivery-mode-of-log-messages-from-container-to-log-driver
* https://docs.docker.com/compose/compose-file/#fragments

> The non-blocking message delivery mode prevents applications from blocking due to logging back pressure. Applications are likely to fail in unexpected ways when STDERR or STDOUT streams block.

While not an issue now, I figured to call this out early in case it does become an issue since we're running a few microservices and databases on the same node.

If we merge this change today, we would:

* no longer have to write to `/etc/docker/daemon.json`
* would ensure all containers would not block on disk contention when attempting to write logs
* would provide all containers with a 100m log buffer should disk contention occur

---- 

UPDATE:

While testing this PR on both staging and prod, it's clear that this doesn't fix the main issue plaguing Content Node disks right now but it does seem like a good additional nonetheless. 

The values kept as defaults (but explicitly added) are:
* `max-size: 10m`
* `max-file: 3`
* `driver: json-file`

The values changes are:
* `mode: non-blocking`
* `max-buffer-size: 100m`

### CODE REVIEW

**Review by jonaylor89 — APPROVED** [review]
LGTM

**Comment by cheran-senthil** [comment]
Anything that is not tested should not be merged, please move this to common services instead

**Comment by SidSethi** [comment]
> Anything that is not tested should not be merged, please move this to common services instead

@cheran-senthil - joaquin wasn't planning on merging this without testing. he just opened it as a way to show what he was recommending, this will definitely be tested first
what does that have to do with your suggestion about common services?

File: creator-node/docker-compose.yml [hunk 1]
```diff
@@ -20,6 +28,13 @@ services:
       service: base-cache
     networks:
       - creator-node-network
+    logging:
+      options:
+        max-size: 10m
+        max-file: 3
+        mode: non-blocking
+        max-buffer-size: 4m
+      driver: json-file
```
Comment (SidSethi) [inline creator-node/docker-compose.yml:38]: noting - max-size = 10m and max-file = 3 seem to be the default, same with `json-file` driver

**Comment by cheran-senthil** [comment]
Again, move this to common services if you plan to merge this in the future

**Comment by jonaylor89** [comment]
> Again, move this to common services if you plan to merge this in the future

I did - I want to merge this asap

**Review by SidSethi — APPROVED** [review]
good with me, pending cheran approval

**Review by cheran-senthil — APPROVED** [review]
Thanks for the changes

**Pull request merged**